### PR TITLE
Feat(style): match with tc 39 website

### PR DESCRIPTION
**Pull request by AugustinMauroy**

### CODE REVIEW

File: css/elements.css [hunk 1]
```diff
@@ -1,9 +1,11 @@
+@import url(https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600&amp;family=Source+Sans+Pro:wght@300;400;600&amp;display=swap);
```
Comment (ljharb) [inline css/elements.css:1]: i'm not sure we want a live dependency on google's font API

Comment (AugustinMauroy) [reply css/elements.css:1]: Do you think that having the font downloaded in the documents would be better?

**Comment by bakkot** [comment]
Thanks for the suggestion, but I don't think we want to change the styling at this time.

File: css/elements.css (hunk 1 above)
Comment (ljharb) [reply css/elements.css:1]: Certainly hosting it ourselves would be better, but i'm not sure how that'd work for all the proposal repos this serves. What's wrong with the current font for specs?

**Pull request closed without merging**